packages/components/psammead-navigation/src/index.jsx

```diff
@@ -157,45 +214,103 @@ const StyledNav = styled.nav`
   }
 `;
 
-const Navigation = ({ children, dir }) => (
-  <StyledNav role="navigation" dir={dir}>
-    <NavWrapper>{children}</NavWrapper>
-  </StyledNav>
-);
+const SwipeableNav = styled.div`
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    white-space: nowrap;
+    overflow-x: scroll;
+    scroll-behavior: smooth;
+    -webkit-overflow-scrolling: touch;
 
-Navigation.propTypes = {
-  children: node.isRequired,
-  dir: oneOf(['ltr', 'rtl']),
-};
+    /* Hide scrollbar */
+    scrollbar-width: none;
+    -ms-overflow-style: none;
+    &::-webkit-scrollbar {
+      display: none;
+    }
 
-Navigation.defaultProps = { dir: 'ltr' };
+    ${({ overflowed }) =>
+      overflowed &&
+      css`
+        &:before {
+          content: ' ';
+          height: 100%;
+          width: 3rem;
+          position: absolute;
+          ${({ dir }) => css`
+            ${dir === 'ltr' ? 'right' : 'left'}: 0;
+          `}
+          bottom: 0;
+          z-index: 3;
+          overflow: hidden;
+          pointer-events: none; /* ignore clicks */
+          background: linear-gradient(
+            ${({ dir }) => (dir === 'ltr' ? 'to right' : 'to left')},
+            ${C_POSTBOX_TRANSPARENT} 0%,
+            ${C_POSTBOX_OPAQUE} 100%
+          );
+        }
+      `}
+  }
+`;
 
-NavigationUl.propTypes = {
-  children: node.isRequired,
-};
+function useOverflowedNav(ref, width) {
+  const [isOverflowed, setIsOverflowed] = useState();
 
-NavigationLi.propTypes = {
-  children: node.isRequired,
-  url: string.isRequired,
-  script: shape(scriptPropType).isRequired,
-  active: bool,
-  currentPageText: string,
-  service: string.isRequired,
-};
+  useEffect(() => {
+    const overflow = ref.current.scrollWidth > ref.current.offsetWidth;
+    setIsOverflowed(overflow);
+  }, [width]);
 
-NavigationLi.defaultProps = {
-  active: false,
-  currentPageText: null,
-};
+  return isOverflowed;
+}
 
-CurrentLink.propTypes = {
-  children: string.isRequired,
-  script: shape(scriptPropType).isRequired,
-  currentPageText: string,
+function useWindowWidth() {
+  const [width, setWidth] = useState(window.innerWidth);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setWidth(window.innerWidth);
+    };
+    window.addEventListener('resize', handleResize);
```

Comment:
Could we replace this with a media query listener? https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia Then we can update the value of `isSwipeable` when that event fires, rather than handling every resize event?

Reply:
Looking at this now, but I am not sure if we can get rid of the resize event as we need that the `useOverflowed` effect is run every time we resize the window, passing the `width`.

Reply:
I have created another custom hook `useMediaQuery` which uses the `matchMedia()` method to determine if the document matches the media query (`max-width: 600px`) https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia and, so we can then pass the boolean value to the `isSwipeable`. 

On the other hand, I am not sure if we can get rid of the resize event as we need that the `useOverflowed` effect is run every time we resize the window, passing the width.